src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -230,6 +230,18 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 ins = INS_crc32cx;
                 break;
 
+            case NI_AdvSimd_ExtractAndNarrowLow:
+                emitSize = EA_8BYTE;
```

Comment:
It would seem like we could get the `emitSize` and the `baseType` used for `opt` from the return type instead?

Reply:
Yeah which is what I wanted to do, but couldn't see anyway to get the return type. I looked into `HWIntrinsic` which defines `baseType` but not `retType` or I didn't see it.. I only see the base type and the operands.

Reply:
If you don't set `HW_Flag_BaseTypeFromFirstArg` or `HW_Flag_BaseTypeFromSecondArg` then it will get the `simdSize` and `baseType` from the return type of the method.

I didn't see anything in particular that needed to know the `simdSize`/`baseType` of the arguments, so I would expect that this "just works".

Reply:
Ah ok, that should make this a lot simpler then, Thanks!.

Reply:
Will we need the specialized codegen logic here at all if you use, as Tanner suggested, `simdSize` and `baseType` of the return value?

Reply:
No we don't. Updated.